source/extensions/all_extensions.bzl

```diff
@@ -37,7 +42,7 @@ _http_filter_prefix = "envoy.filters.http"
 def envoy_all_http_filters():
     all_extensions = dicts.add(_required_extensions, EXTENSIONS)
 
-    return [v for k, v in all_extensions.items() if k.startswith(_http_filter_prefix)]
+    return [v for k, v in all_extensions.items() if k.startswith(_http_filter_prefix) and k != _kill_request_http_filter]
```

Comment:
Can you refactor so we only have to add the filter once to the denylist?

Reply:
I might be wrong, but I don't see a way to refactor to make it simpler since denylist is a function parameter in `envoy_all_extensions()`. We always need some logic in `envoy_all_http_filters` to filter out kill_request_http_filter

Reply:
Just create a `GLOBAL_DENYLIST` definition at the top and use it in both places.

Reply:
Done